### ISSUE TITLE
Change the default tree name from default to something else

### ISSUE THREAD

**Issue by KonstantinSchubert** [scikit-hep/root_pandas#32]
The trouble with `default` as a tree name is that the ROOT interpreter reads it as the C++ `default` key word and throws an error.

**Comment by ibab** [comment]
Woah. Can you give an example of when the error occurs?

**Comment by KonstantinSchubert** [comment]
Well, here is what is looks like:
```

[mteal@bam ~]$ root
   ------------------------------------------------------------
  | Welcome to ROOT 6.04/03                http://root.cern.ch |
  |                               (c) 1995-2014, The ROOT Team |
  | Built for linuxx8664gcc                                    |
  | From tag v6-04-02-110-g264f10d, 14 July 2015               |
  | Try '.help', '.demo', '.license', '.credits', '.quit'/'.q' |
   ------------------------------------------------------------

root [0] TBrowser r
(TBrowser &) @0x7f468caa6028
root [1] (class TFile *) 0x6723cd0
(class TFile *) 0x71291c0
(class TFile *) 0x742b620

root [1]
root [1] default->Draw("Ds_M")
ROOT_prompt_1:1:8: error: expected ':'
      after 'default'
default->Draw("Ds_M")
       ^
       :
ROOT_prompt_1:1:8: error: expected
      expression
ROOT_prompt_1:1:1: error: 'default'
      statement not in switch statement
default->Draw("Ds_M")
^
root [2]
```

**Comment by ibab** [comment]
I see. So you can't use the variable that is automatically created by `TBrowser`.
I don't think changing the name would cause problems with backwards compatibility.
Which name would you suggest?

**Comment by KonstantinSchubert** [comment]
I am not sure if it would actually not cause problems with backwards compatibility. Maybe "tree" is a good name?

What do you mean with "you can't use the variable that is automatically created by TBrowser" ? Are you saying that he TBrowser also uses `default` as a name?

**Comment by ibab** [comment]
In your example, the variable `default` magically appears in the interpreter.
I thought this is caused by `TBrowser` defining it.

**Comment by KonstantinSchubert** [comment]
Ah, sorry. I think what happened is that Mikey opended the TBrowser, then clicked on a ROOT file which had a tree named `default` in it. I will ask him to create a better example.

**Comment by KonstantinSchubert** [comment]
Better example:
```
[mteal@bam blinded]$ root -l tau25mu_MC_2012.root
root [0]
Attaching file tau25mu_MC_2012.root as _file0...
(class TFile *) 0x61f4600
root [1] default->Draw("tau_M")
ROOT_prompt_1:1:8: error: expected ':' after 'default'
default->Draw("tau_M")
       ^
       :
ROOT_prompt_1:1:8: error: expected expression
ROOT_prompt_1:1:1: error: 'default' statement not in switch statement
default->Draw("tau_M")
^
root [2]
```

**Comment by ibab** [comment]
Does this happen with all ROOT versions, or just with a new ROOT 6?

**Comment by KonstantinSchubert** [comment]
I can confirm it does not happen with 5.34/24

**Comment by chrisburr** [comment]
Resolved with #42.
